src/pages/Trade/Trade.tsx

```diff
@@ -119,28 +136,19 @@ export const Trade = memo(() => {
         gap={4}
       >
         <FormProvider {...methods}>
-          <Switch location={location}>
-            <Route key={LimitOrderRoutePaths.Input} path={LimitOrderRoutePaths.Input}>
-              <LimitOrder
-                tradeInputRef={tradeInputRef}
-                onChangeTab={handleChangeTab}
-                isRewritingUrl={isRewritingUrl}
-                defaultBuyAssetId={defaultBuyAssetId}
-                defaultSellAssetId={defaultSellAssetId}
-              />
-            </Route>
-            <Route key={ClaimRoutePaths.Select} path={ClaimRoutePaths.Select}>
-              <Claim onChangeTab={handleChangeTab} />
-            </Route>
-            <Route key={TradeRoutePaths.Input} path={TradeRoutePaths.Input}>
-              <MultiHopTrade
-                isRewritingUrl={isRewritingUrl}
-                defaultBuyAssetId={defaultBuyAssetId}
-                defaultSellAssetId={defaultSellAssetId}
-                onChangeTab={handleChangeTab}
-              />
-            </Route>
-          </Switch>
+          <Routes>
+            <Route
+              key={LimitOrderRoutePaths.Input}
+              path={LimitOrderRoutePaths.Input}
+              element={limitOrderElement}
+            />
+            <Route
+              key={ClaimRoutePaths.Select}
+              path={ClaimRoutePaths.Select}
+              element={claimElement}
+            />
+            <Route key={TradeRoutePaths.Input} path={''} element={tradeElement} />
```

Comment:
q: do we really want this empty path hovef `parth='/'`? (considering we are under react-router umbrella)

Reply:
What about: 6114148e82

Reply:
That's even better